jwtauth.go

```diff
@@ -238,6 +238,31 @@ func SetExpiryIn(claims map[string]interface{}, tm time.Duration) {
 	claims["exp"] = ExpireIn(tm)
 }
 
+// Set subject ("sub") in the claims
+func setSub(claims map[string]interface{}, sub string) {
+	claims["sub"] = sub
+}
+
+// Set audience ("aud") in the claims
+func setAudience(claims map[string]interface{}, aud string) {
+	claims["aud"] = aud
+}
+
+// Set JWT ID ("jti") in the claims
+func setJwtId(claims map[string]interface{}, jti string) {
+	claims["jti"] = jti
+}
+
+// Set not Before ("nbf") in the claims
+func setNotBefore(claims map[string]interface{}, nbf time.Time) {
+	claims["nbf"] = nbf.UTC().Unix()
+}
+
+// Add custom additional claim in the claims (For private/public claims, RFC 7519)
+func setXClaim(claims map[string]interface{}, claimName string, claimValue interface{}) {
+	claims[claimName] = claimValue
+}
```

Comment:
Since these new functions are not exported, what's the point of adding it to the codebase?

Reply:
Please check the new edits